src/main/java/org/opensearch/geospatial/index/mapper/xypoint/XYPoint.java

```diff
@@ -132,13 +145,14 @@ public XYPoint resetFromCoordinates(String value, final boolean ignoreZValue) {
      * @param value  input String which contains WKT POINT that needs to be parsed and validated
      * @param ignoreZValue  boolean parameter which decides if z coordinate needs to be ignored or not
      * @return XYPoint after setting the x and y coordinates
+     * throws OpenSearchParseException
      */
     private XYPoint resetFromWKT(String value, boolean ignoreZValue) {
         Geometry geometry;
         try {
             geometry = new WellKnownText(false, new StandardValidator(ignoreZValue)).fromWKT(value);
         } catch (Exception e) {
-            throw new OpenSearchParseException("Invalid WKT format", e);
+            throw new OpenSearchParseException("Invalid WKT format, [{}]", value);
```

Comment:
You have missed the exception trace. Please add the exception trace with the exception you are throwing here.

Reply:
I removed it intentionally because the error is same with or without adding it
```
{
    "error": {
        "root_cause": [
            {
                "type": "parse_exception",
                "reason": "Invalid WKT format, [POINTSTEST (-71.34 41.12)]"
            }
        ],
        "type": "mapper_parsing_exception",
        "reason": "failed to parse field [geometry] of type [xy_point]",
        "caused_by": {
            "type": "parse_exception",
            "reason": "Invalid WKT format, [POINTSTEST (-71.34 41.12)]"
        }
    },
    "status": 400
}

Reply:
The error message will be same but the you will loose the error trace. So, add that.

Reply:
Got it, updated